buddysuite/buddy_resources.py

```diff
@@ -1932,12 +1932,12 @@ def isfile_override(path):
             "unroot": {"flag": "ur",
                                "action": "store_true",
                                "help": "Remove any roots"},
-            "descending_order": {"flag": "do",
-                                 "action": "store_true",
-                                 "help": "sort the child nodes in order of the number of their children, in decending order"},
-            "ascending_order": {"flag": "ao",
-                                 "action": "store_true",
-                                 "help": "sort the child nodes in order of the number of their children, in ascending order"},
+            "ladderize": {"flag": "ld",
```

Comment:
This should actually live between "hash_ids" and "list_ids" to keep alpha-order. Apparently hash_ids and list_ids are themselves out of order, so swap them around too.